src/LookingGlassConfig.ts

```diff
@@ -103,6 +106,20 @@ export type ViewControlArgs = {
 	 * @default InlineView.Center
 	 */
 	inlineView: InlineView
+	/**
```

Comment:
This exposes a reference to the `popup window`, a reference to the `capturing` state, and a reference to the current `XRSession`. Currently the popup window reference is used in code, but `capturing` and `XRSession` are not. These references are to prepare for future functionality.